### PR TITLE
New italian translation

### PR DESCRIPTION
**Pull request by filipporomani**
Everything should be updated now.

### CODE REVIEW

**Comment by staniel359** [comment]
@filipporomani Please update your local repo.

File: shared/locales/it.json [hunk 1]
```diff
@@ -37,7 +37,6 @@
     "posts": "Post",
     "conversations": "Conversazioni",
     "library": "Libreria",
-    "recommendations": "Suggerimenti",
```
Comment (staniel359) [inline shared/locales/it.json:40]: @filipporomani This one shouldn't be removed.
The actual one is on line 537.

File: shared/locales/it.json [hunk 2]
```diff
@@ -121,6 +120,7 @@
     "add": "Aggiungi",
     "create": "Crea",
     "edit": "Modifica",
+    "filter": "Filtra",
```
Comment (staniel359) [inline shared/locales/it.json:123]: @filipporomani Please keep the original order of the keys.

**Review by staniel359 — CHANGES_REQUESTED** [review]
@filipporomani Please take a look.

**Comment by staniel359** [comment]
@ZinRicky Hello.
How's your computer?

**Comment by staniel359** [comment]
@starise Hello. Would you like to help with the translation?

**Comment by staniel359** [comment]
@filipporomani Hello.
Could you take a look at my notes?

**Comment by ZinRicky** [comment]
> @ZinRicky Hello. How's your computer?

Not well, unfortunately. The mobo short-circuited, and the bill is ~1000€, which I don’t have at the moment.

**Comment by staniel359** [comment]
@ZinRicky Too bad.
So you won't be able to complete the translation?

**Comment by ZinRicky** [comment]
I’m afraid I won’t. I don’t have a PC of mine at disposal

**Comment by filipporomani** [comment]
Hello, currently I'm a bit confused about the modifications to do. I almost never worked on GH PRs and I don't understand the comparison.

**Comment by filipporomani** [comment]
Hello, currently I'm a bit confused about the modifications to do. I almost never worked on GH PRs and I don't understand the comparison.

**Comment by staniel359** [comment]
@ZinRicky I see.
Well, I hope you'll solve your problem.

**Comment by staniel359** [comment]
@filipporomani Okay.
Then could you just create another PR?
There are no more modifications so far.

**Comment by staniel359** [comment]
@filipporomani Just wanted to release a new version, and I'd really appreciate if you complete Italian translation.

**Comment by staniel359** [comment]
@filipporomani Just wanted to release a new version, and I'd really appreciate if you complete Italian translation.

**Comment by filipporomani** [comment]
Created #79, refer to that from now.

**Comment by staniel359** [comment]
@ZinRicky Hello.
If your don't have a computer yet, is there a chance that you have mobile Discord?
So you could help me with translations there.

**Comment by staniel359** [comment]
@ZinRicky Hey, how's it going?

**Pull request closed without merging**